### PR TITLE
chore: mistyped extension to lowercase

### PR DESCRIPTION
**Pull request by namnm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,4 +43,4 @@ To start in development mode, run
 yarn start
 ```
 
-To contribute to the source, please read [CONTRIBUTING.md](https://github.com/PastelNetwork/Pastel_Electron_Wallet/blob/master/docs/CONTRIBUTING.MD).
+To contribute to the source, please read [CONTRIBUTING.md](https://github.com/PastelNetwork/Pastel_Electron_Wallet/blob/master/docs/CONTRIBUTING.md).
```
Comment (br4in3x) [inline README.md:46]: If you try to navigate to https://github.com/PastelNetwork/Pastel_Electron_Wallet/blob/master/docs/CONTRIBUTING.md in the browser, you will see that this link is broken. The previous one was correct (with an uppercase extension).

Comment (namnm) [reply README.md:46]: This PR also changes the filename from uppercase to lowercase. The link should work after merged.

**Review by Dicklesworthstone — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (br4in3x) [reply README.md:46]: @namnm Sorry it took long, I don't think the above is true. If you look at the file structure of this branch (https://github.com/pastelnetwork/pastel-electron-wallet/tree/hotfix/mistyped-extension-contributing/docs), you will see that `CONTRIBUTING.MD` still has uppercase extension

Comment (namnm) [reply README.md:46]: @br4in3x Thanks, I have just used `git mv` to perform the update

**Review by br4in3x — APPROVED** [review]

**Comment by br4in3x** [comment]
@namnm please feel free to merge :)

**Pull request merged**